Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityControllerMappingProfileInspector.cs

```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.﻿
 
```

Comment:
Another bug I didn't find a line for.

Repro steps:

1. Scroll down to the generic Open VR Controllers. Do this on the **Left** Hand Controller.
2. Try to change the Generic Type to Unity Controller while there's a Handedness selected. It won't change.
3. Switch the handedness to None, and switch to Unity Controller.
4. Observe that the Right Hand Controller has been changed to Unity Controller instead of the Left Hand Controller.

Reply:
Great catch, fixed.